shardingsphere-proxy/shardingsphere-proxy-backend/src/main/java/org/apache/shardingsphere/proxy/backend/text/admin/mysql/executor/UseDatabaseExecutor.java

```diff
@@ -48,7 +47,6 @@ public void execute(final BackendConnection backendConnection) {
     
     private boolean isAuthorizedSchema(final BackendConnection backendConnection, final String schema) {
         Optional<ShardingSpherePrivilege> privilege = ProxyContext.getInstance().getMetaDataContexts().getAuthentication().findPrivilege(backendConnection.getGrantee());
-        // TODO : Need to check whether PrivilegeType.USAGE is correct or enough?
-        return privilege.isPresent() && privilege.get().hasPrivileges(schema, Collections.emptyList());
+        return privilege.isPresent() && privilege.get().hasPrivileges(schema) ? true : false;
```

Comment:
return privilege.isPresent() && privilege.get().hasPrivileges(schema)

Reply:
@tristaZero It has been changed.